moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_objects.cpp

```diff
@@ -73,17 +73,17 @@ QString subframe_poses_to_qstring(const moveit::core::FixedTransformsMap& subfra
 
 namespace moveit_rviz_plugin
 {
-void MotionPlanningFrame::importFileButtonClicked()
+void MotionPlanningFrame::import3DObjectFromFileButtonClicked()
 {
-  QString path = QFileDialog::getOpenFileName(this, tr("Import Object"));
+  QString path = QFileDialog::getOpenFileName(this, tr("Import 3D Object (e.g. .stl, "));
```

Comment:
I suggest [using file filters](https://doc.qt.io/qt-5/qfiledialog.html#getOpenFileName):
```suggestion
  QString path = QFileDialog::getOpenFileName(this, tr("Import Object Mesh"), QString(), "CAD files (*.stl *.obj)");
```

Reply:
Good idea, but we would have to maintain the lists of available file formats though. Do we want that?

Reply:
As a fallback, you can still provide an "all files (*.*)" filter.

Reply:
In that case is there a difference to not touching this at all?

Reply:
Done